### PR TITLE
add docs for setup on Archlinux

### PR DESCRIPTION
**Pull request by vdloo**
This should answer some of the questions we receive occasionally.

### CODE REVIEW

File: Documentation/archlinux_setup.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+This document is a step by step guide on how to set up hypernode-vagrant on a fresh [Arch Linux](https://www.archlinux.org/) installation. Getting the box to run on Arch Linux is a bit more laborious than getting it to run on Ubuntu because on Arch you have to set up the [lxc](https://linuxcontainers.org/) container's networking yourself.
+
+Note that these instructions assume that the user is root and that the lxc is not an unprivileged container. To set it up for less privileged users take not of [the vagrant-lxc sudo wrapper](https://github.com/fgrehm/vagrant-lxc/blob/97b5882262fcc5c652123c88515b8f925c42f574/lib/vagrant-lxc/sudo_wrapper.rb#L10) and keep an eye on [this thread about using unprivileged containers](https://github.com/fgrehm/vagrant-lxc/issues/312).
```
Comment (kerrermanisNL) [inline Documentation/archlinux_setup.md]: *take note of

File: Documentation/archlinux_setup.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+### Log in to the container
+
+If you did not have an SSH agent running you should run:
+```bash
+# Log in as the default vagrant user
+vagrant ssh
+
+# And then inside the container:
+
```
Comment (kerrermanisNL) [inline Documentation/archlinux_setup.md]: Is there something missing here?

Comment (vdloo) [reply Documentation/archlinux_setup.md]: no but I removed the newline now for clarity

File: Documentation/archlinux_setup.md (hunk 1 above)
Comment (vdloo) [reply Documentation/archlinux_setup.md]: fixed

**Pull request merged**